helpers/codegen/codegen.js

```diff
@@ -48,6 +49,7 @@ export const codegens = [
   SalesforceApexCodegen,
   ShellHTTPie,
   WgetCodegen,
+  NodeUnirestRequestCodegen,
```

Comment:
Change `NodeUnirestReq`  to `NodeJsUnirestReq`.
Also we keep alphabetical ordering in this array of codegens. So move unirest right next to `NodeJsNativeCodegen`.

Reply:
Also rebase to latest `master` branch.

Reply:
@liyasthomas above mentioned feedbacks are encorporated.